lib/contentful.js

```diff
@@ -74,53 +79,57 @@ module.exports.getClient = function getClient() {
  */
 module.exports.contentfulError = function contentfulError(error) {
   const scope = error;
-  scope.message = `Contentful: ${error.message}`;
+  scope.message = `${ERROR_PREFIX} ${error.message}`;
 
   return scope;
 };
 
+module.exports.getFirstItemFromArray = function getFirstItemFromArray(array) {
```

Comment:
I though about using underscore actually. I though It wasn't necessary to bring another library for this. If we start needing more methods, like the _.extend, I think it makes sense to just include it.